hips/tiles/description.py

```diff
@@ -2,25 +2,37 @@
 from collections import OrderedDict
 
 __all__ = [
-    'HipsDescription',
+    'HipsSurveyProperties',
 ]
 
 
-class HipsDescription:
+class HipsSurveyProperties:
     """HiPS properties container.
 
     Parameters
     ----------
     properties : `~collections.OrderedDict`
-        HiPS description properties
+        HiPS survey properties
+
+    Examples
+    --------
+    ::
+
+        >>> import urllib.request
+        >>> response = urllib.request.urlopen('https://raw.githubusercontent.com/hipspy/\
```

Comment:
This docs example is failing at the moment:
https://gist.github.com/cdeil/422a87d598b1e60e7e5c6bff25d1b716

Please add a `fetch` method to this `HipsSurveyProperties` class and call it from the docs example.